CorsixTH/Lua/hospital.lua

```diff
@@ -1614,6 +1614,18 @@ function Hospital:humanoidDeath(patient)
   end
 end
 
+--! Handyman has died. Remove all tasks assigned to this staff
+--!param handyman The handyman deceased
+function Hospital:handymanDeath(handyman)
+  for i = 1, #self.handymanTasks do
```

Comment:
There is a function buried in the code called removeHandymanTask. Is there a chance it looks similar to this?